README.md

```diff
@@ -236,6 +237,51 @@ up-to-date. If the lock file is missing, or it needs to be updated, cargo-upgrad
 error. If the '--to-lockfile' flag is supplied then the network won't be accessed.
 ```
 
+### `cargo set-version`
+
+Set the version in your `Cargo.toml`.
+
+#### Examples
+
+```sh
+# Set the version to the version 1.0.0
+$ cargo set-version 1.0.0
+# Bump the version to the next major
+$ cargo set-version --bump major
+# Bump version to the next minor
+$ cargo set-version --bump minor
+# Bump version to the next patch
+$ cargo set-version --bump patch
+```
+
+#### Usage
+
+```plain
+cargo-set-version 0.7.0
+Change a package's version in the local manifest file (i.e. Cargo.toml)
+
+USAGE:
+    cargo set-version [FLAGS] [OPTIONS] [--] [target]
+
+FLAGS:
+        --all          [deprecated in favor of `--workspace`]
+        --dry-run      Print changes to be made without making them
+    -h, --help         Prints help information
+    -V, --version      Prints version information
+        --workspace    Modify all packages in the workspace
+
+OPTIONS:
+        --bump <bump>             Increment manifest version [possible values: major, minor, patch,
+                                  release, rc, beta, alpha]
+        --exclude <exclude>...    Crates to exclude and not modify
+        --manifest-path <path>    Path to the manifest to upgrade
+    -m, --metadata <metadata>     Version metadata
```

Comment:
is that what goes after `+` https://docs.rs/semver/1.0.4/semver/struct.BuildMetadata.html
would be nice to provide an example

Reply:
Good idea.  I gave an example of a use case and in the README pointed to that doc since it does such a good job describing it and how one might use it.  Figure it'd be better to link than having to maintain our own copy.